stable/traefik/README.md

```diff
@@ -145,6 +145,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `deployment.hostPort.httpEnabled`      | Whether to enable hostPort binding to host for http.          | `false`                                   |
 | `deployment.hostPort.httpsEnabled`     | Whether to enable hostPort binding to host for https.         | `false`                                   |
 | `deployment.hostPort.dashboardEnabled` | Whether to enable hostPort binding to host for dashboard.     | `false`                                   |
+| `sendAnonymousUsage`            | Send anonymous usage statistics.                                     | `false`                                   |
```

Comment:
This should explicitly be added to `values.yaml` as well. I know the default value will implicitly be `false`, but as indicated by another comment, I believe many people learn about available options from `helm inspect values stable/traefik` rather than by perusing the README.

Reply:
Fixed and squashed into commit for this one as well.